### DISCUSSION TITLE
The new website, the studio, the SDK

### DISCUSSION THREAD

**Discussion by lexoyo** [GrapesJS/grapesjs#6361] [General]
Hello @artf 

I love the  the new website, it looks neat, congrats!

Sorry to ask but it's now showing and linking to the studio and it says "embeddable drag-and-drop builder"

1. This site is not about the framework anymore then right? But the framework will live on?
1. The studio is not open source, right? But the title says it is open source, I am confused
1. I link to this page from many places saying it's an open source framework, do I have to update it? Should I link to github then?

**Comment by artf** [comment]
Hi Alex,

Thanks for the kind words and for bringing this up—sorry for any confusion caused. You're correct that the website mixes content about the GrapesJS framework (open-source) and the Studio SDK (commercial). The focus has shifted a bit more toward the Studio SDK intentionally, but rest assured, the GrapesJS framework remains free and open-source.

If you're linking to showcase GrapesJS as an open-source framework, I'd recommend directing users to the GitHub repo to avoid any misunderstandings.

**Reply by lexoyo** [reply]
Ok
